docker/install-scrypted-dependencies-win.ps1

```diff
@@ -44,30 +52,19 @@ var svc = new Service({
     },
   ]
 });
-
-svc.on('alreadyuninstalled', () => {
-  svc.install();
-});
-
-svc.on('uninstall', () => {
-  svc.install();
-});
-
-svc.on('error', () => {
-  svc.install();
-});
-
 svc.on('install', () => {
-  svc.start();
+  console.log("Service installed");
 });
-
-svc.uninstall();
```

Comment:
I found this quite redundant so simplified, not sure if there was a reason for all the other event listeners.

Reply:
This code would trigger an install after uninstalling it, or attempting, first. The intention is to make sure the service script can update itself.

Reply:
Ah ok. There seems to be a bug or something with `node-windows` uninstall at the moment that causes an exception https://github.com/coreybutler/node-windows/issues/324 even though the service is successfully uninstalled.

Reply:
Ah figured out a workaround.